Ghost DataHUD toggle now shows all HUDs at once

GunHog opened the pull request:
Ghost Data HUD toggle switched to show all HUDs at once!

DataHUDs were redesigned such that they no longer conflict, therefore it is entirely safe to use them all.

:cl: Name: Gun Hog
tweak: Security/Medical/Diagnostic HUD toggle for ghosts now combines all three HUD types into one!
rscdel: Security HUD removed from Antag HUD due to conflicts with the ghost toggle.
/:cl:

![screenshot 2016-04-16 15 21 50](https://cloud.githubusercontent.com/assets/4955510/14583508/61463e5e-03e9-11e6-99f0-46a3d155c156.png)

Commit:
Show all HUDs at once

Ghost Data HUD toggle switched to show all HUDs at once.

ChangelingRain commented:
>making ghosts able to judge a situation effectively at any time

ghost powercreep

lzimann commented:
Will it conflict with the admin antag hud? Since admin hud also gives sec hud.

ChangelingRain commented:
Sort of. The admin antag hud toggles based on the state of the security hud, which is really dumb; if you have the ghost hud on, it won't trigger antag hud, and will instead toggle ghost sec hud off.

GunHog commented:
Wait, what? I did not know that about admin antag HUDs, good catch. I will have to fish up that code and see what I need to do about it.

GunHog commented:
So after thinking about it and writing a lot of nasty snowfake to check each other for activeness, I have settled on the simplest solution: Removing the Security HUD from the Antag HUD toggle. This should allow them to co-exist without conflict.

Opinions?

ChangelingRain commented:
I'd say remove the sechud yeah, i'm not sure what other admins would say but for me it's just mildly annoying

lzimann commented:
I personally like it, but if you can activate it with the ghost hud along with antag hud, whatever by me.

Commit:
Removes SecHUD from AntagHUD

The SecHUD has been seperated from the Antag HUD. Admin ghosts can get
this function through the normal Ghost HUD toggle.

GunHog commented:
Fix uploaded, the WIP tag may now be removed, this is ready.
